contrib/freeze_packages.sh

```diff
@@ -6,8 +6,8 @@ contrib=$(dirname "$0")
 
 which virtualenv > /dev/null 2>&1 || { echo "Please install virtualenv" && exit 1; }
 
-rm $venv_dir -rf
-virtualenv $venv_dir
+rm "$venv_dir" -rf
+python3 -m virtualenv $venv_dir
```

Comment:
Please change this to use e.g. -p $(which python3) as that seems to be the preferred way to do this.

Reply:
Not sure about that. But as you wish.